CHANGES/7180.bugfix

```diff
@@ -0,0 +1 @@
+`ConnectionResetError` will always be raised when `StreamWriter.write` is called after `connection_lost` has been called on the `BaseProtocol`
```

Comment:
Remember to use RST syntax here, not markdown.

Reply:
https://github.com/aio-libs/aiohttp/tree/master/CHANGES#examples-for-adding-changelog-entries-to-your-pull-requests

Reply:
860fde8aea3c78b3c5a769c38ce8308ac6787d22